modules/statistics/js/form_stats_behavioural.js

```diff
@@ -1,10 +1,11 @@
 /*global document, $, window, scrollContent*/
 function updateBehaviouralTab() {
     var BehaviouralProject = document.getElementById("BehaviouralProject");
+    var BehaviouralSubproject = document.getElementById("BehaviouralSubproject");;
     var request            = $.ajax(
         {
-            url: loris.BaseURL + '/statistics/stats_behavioural/?dynamictabs=dynamictabs&BehaviouralProject=' + (BehaviouralProject==null ? "" : BehaviouralProject.value),
-            type: 'GET',
+            url: loris.BaseURL + '/statistics/stats_behavioural/?dynamictabs=dynamictabs&BehaviouralProject=' + (BehaviouralProject==null ? "" : BehaviouralProject.value) + '&BehaviouralSubproject=' + (BehaviouralSubproject==null ? "" : BehaviouralSubproject.value),
+            type: 'POST',
```

Comment:
Why is this changed to a POST? (Especially since the things that are added are GET parameters, not posted..)